.travis.yml

```diff
@@ -22,6 +22,7 @@ env:
     matrix:
         - JOB_TYPE=compile_and_basic_tests
         - JOB_TYPE=acceptance_tests
+        - JOB_TYPE=backend_integration_tests
```

Comment:
I would suggest switch to build stages especially due to Jenkins trigger.

Something around this setup (each layer is concurrent, all jobs in higher layer have to pass successfully)
1) job:compile_and_basic_tests
2) job:acceptance_tests | job:backend_integration_tests
3) trigger jenkins

This way we make raw-code-style / signature / changelog tests before we start anything more serious.
We do not trigger jenkins until everythign else is passing successfully. 

Example implementation: https://github.com/mendersoftware/tenantadm/pull/29/files

Reply:
when multiple "stages" have same name they are concurrent